onchain/rollups/test/foundry/portals/ERC20Portal.t.sol

```diff
@@ -1,408 +1,160 @@
 // (c) Cartesi and individual authors (see AUTHORS)
```

Comment:
Yes, these types of unit tests are much easier to maintain since they don't depend on specific token implementations from OpenZeppelin, which might change over time.
However, I do reckon we should also have integration tests of portals + OpenZeppelin implementations for ERC-20, ERC-721 and ERC-1155.
Let's leave this to a later PR, shall we?